docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.9"
+services:
+  broker:
+    image: rabbitmq
+    ports:
+      - "5672:5672"
+
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_DB=antiplag
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - "5432:5432"
+    volumes:
+      - database-data:/var/lib/postgresql/data/
+
+  web:
+    build: .
+    image: antiplag/backend:v1
+    volumes:
+      - .:/backend/antiplag
+    command: pipenv run bash -c "cd antiplag && python manage.py runserver 0.0.0.0:8000"
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+
+  worker:
```

Comment:
Locally, me and @Hemii have troubles on running the worker. 
Here's the error log, could you look at it and test it properly?

![image](https://user-images.githubusercontent.com/27869979/109552850-97986580-7ad2-11eb-9d51-77f54c0cfcb2.png)

Reply:
It takes few seconds to spin up broker server, hence the errors. However I only see maximum of 1-2 tries. I suggest we keep this as is and see if this problem persists.

Reply:
I agree.